packages/javascript/FlatfileJavascript.ts

```diff
@@ -179,20 +176,34 @@ function initializeIFrameConfirmationModal(
   confirmModal.style.display = 'none'
   document.body.appendChild(confirmModal)
 
-  window.addEventListener(
-    'message',
-    (event) => {
+  function closeFlatfileSpace(event: any) {
```

Comment:
We should likely use a similar re-usable pattern like we do in `handlePostMessage from @flatfile/embedded-utils`. Can be a fast follow!

Reply:
Done in #135 @bangarang !